### PR TITLE
chore: konflux: remove TODO

### PR DESCRIPTION
**Pull request by psturc**
[KFLUXBUGS-1185](https://issues.redhat.com//browse/KFLUXBUGS-1185)

### CODE REVIEW

File: ci-operator/step-registry/redhat-appstudio/load-test/infra-deployments/redhat-appstudio-load-test-infra-deployments-commands.sh [hunk 1]
```diff
@@ -91,8 +91,6 @@ echo "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com" >"${GIT_CREDS_PATH}"
 cd "$(mktemp -d)"
 
 git clone --origin upstream --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
-# TODO this change is needed only for backwards compatibility, remove after https://github.com/redhat-appstudio/e2e-tests is merged
-git remote add origin https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/load-test/infra-deployments/redhat-appstudio-load-test-infra-deployments-commands.sh:95]: /assign @jhutar
this is a follow-up on the previous PR. PTAL, thanks!

**Review by jhutar — APPROVED** [review]
/lgtm

**Comment by jhutar** [comment]
Thank you!

**Pull request merged**